src/components/unique-token/UniqueTokenImage.js

```diff
@@ -62,6 +63,7 @@ const UniqueTokenImage = ({
   const isSVG = isSupportedUriExtension(imageUrl, ['.svg']);
   const newImageUrl = transformSvgs ? svgToPngIfNeeded(imageUrl) : imageUrl;
   const image = isENS && !isSVG ? `${item.image_url}=s1` : newImageUrl;
+  const lowResUrl = isSVG ? newImageUrl : getLowResUrl(image);
```

Comment:
actually, sorry, I think you get this here so it's ok I guess

Reply:
yeah exactly 👌